### PR TITLE
fix(DIS): fix DIS instance resource lint error

### PR DESCRIPTION
**Pull request by luoping-12345**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
fix DIS instance resource lint error

**Special notes for your reviewer**:
testcase error has not been resolved
**Release note**:
```
./scripts/codecheck.sh ./huaweicloud/services/dis

==> Checking for running environment...

==> Applying patch...

==> Checking for code complexity...
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Language                              Files     Lines   Blanks  Comments     Code Complexity Complexity/Lines
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Go                                        1       567       64         8      495         67            13.54
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
~dbss/resource_huaweicloud_dbss_instance.go       567       64         8      495         67            13.54
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Total                                     1       567       64         8      495         67            13.54
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Processed 17738 bytes, 0.018 megabytes (SI)
─────────────────────────────────────────────────────────────────────────────────────────────────────────────

the TOP10 most complex functions:
11 dbss resourceInstanceCreate huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go:187:1
7 dbss createInstaceWaitingForStateCompleted huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go:349:1
6 dbss waitForInstanceDelete huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go:520:1
6 dbss resourceInstanceRead huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go:416:1
4 dbss FilterInstances huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go:483:1
4 dbss buildCreateInstanceBodyParams huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go:281:1
3 dbss resourceInstanceDelete huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go:496:1
1 dbss buildCreateInstanceRequestBodyCreateInstanceRequestBodyProductInfos huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go:337:1
1 dbss buildCreateInstanceRequestBodyCreateInstanceRequestBodySecurityGroups huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go:329:1
1 dbss buildCreateInstanceRequestBodyCreateInstanceRequestBodyNics huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go:320:1
Average: 3.83

==> Checking for golangci-lint...

==> Checking for Nolint directives...

==> Checking for TF features in dbss...

==> Checking for misspell in dbss...

==> Checking for code complexity in ./huaweicloud/services/acceptance/dbss...
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Language                              Files     Lines   Blanks  Comments     Code Complexity Complexity/Lines
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Go                                        1       141       16         1      124          8             6.45
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
~resource_huaweicloud_dbss_instance_test.go       141       16         1      124          8             6.45
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Total                                     1       141       16         1      124          8             6.45
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Processed 4223 bytes, 0.004 megabytes (SI)
─────────────────────────────────────────────────────────────────────────────────────────────────────────────

the TOP5 most complex functions:
5 dbss getInstanceResourceFunc huaweicloud/services/acceptance/dbss/resource_huaweicloud_dbss_instance_test.go:20:1
1 dbss testInstance_basic huaweicloud/services/acceptance/dbss/resource_huaweicloud_dbss_instance_test.go:122:1
1 dbss testInstance_base huaweicloud/services/acceptance/dbss/resource_huaweicloud_dbss_instance_test.go:101:1
1 dbss TestAccInstance_basic huaweicloud/services/acceptance/dbss/resource_huaweicloud_dbss_instance_test.go:59:1
Average: 2

==> Checking for golangci-lint in ./huaweicloud/services/acceptance/dbss...

==> Checking for Nolint directives in ./huaweicloud/services/acceptance/dbss...

==> Cleanup patch...

Check Completed!

```

## PR Checklist

* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/dis" TESTARGS="-run TestAccResourceDisStream_basic"
...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dis -v -run TestAccResourceDisStream_basic -timeout 360m -parallel 4
=== RUN   TestAccResourceDisStream_basic
=== PAUSE TestAccResourceDisStream_basic
=== CONT  TestAccResourceDisStream_basic
--- PASS: TestAccResourceDisStream_basic (40.22s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dis       40.258s
```

### CODE REVIEW

File: huaweicloud/services/dis/resource_huaweicloud_dis_stream.go [hunk 1]
```diff
@@ -174,9 +176,9 @@ func ResourceDisStream() *schema.Resource {
...
 
 func resourceDisStreamCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	config := meta.(*config.Config)
-	region := config.GetRegion(d)
-	client, err := config.DisV2Client(region)
+	conf := meta.(*config.Config)
+	region := conf.GetRegion(d)
+	client, err := conf.DisV2Client(region)
 	if err != nil {
 		return diag.Errorf("error creating DIS v2 client, err=%s", err)
```
Comment (jinyangyang222) [inline huaweicloud/services/dis/resource_huaweicloud_dis_stream.go:183]: return diag.Errorf("error creating DIS v2 client: %s", err)

File: huaweicloud/services/dis/resource_huaweicloud_dis_stream.go [hunk 2]
```diff
@@ -174,9 +176,9 @@ func ResourceDisStream() *schema.Resource {
 }
 
 func resourceDisStreamCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```
Comment (jinyangyang222) [inline huaweicloud/services/dis/resource_huaweicloud_dis_stream.go:178]: func resourceStreamCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {

Comment (jinyangyang222) [reply huaweicloud/services/dis/resource_huaweicloud_dis_stream.go:178]: It is already in the dis package, so there is no need to add dis to the function name, Please check all relevant areas.

File: huaweicloud/services/dis/resource_huaweicloud_dis_stream.go (hunk 1 above)
Comment (jinyangyang222) [reply huaweicloud/services/dis/resource_huaweicloud_dis_stream.go:183]: Pls check all relevant areas.

File: huaweicloud/services/dis/resource_huaweicloud_dis_stream.go [hunk 3]
```diff
@@ -349,7 +351,7 @@ func checkPartitionUpdateResult(ctx context.Context, client *golangsdk.ServiceCl
 			if err != nil {
 				return nil, "failed", err
 			}
-			logp.Printf("WritablePartitionCount=", resp.WritablePartitionCount, targetValue)
+			log.Printf("WritablePartitionCount: %d, targetValue: %d", resp.WritablePartitionCount, targetValue)
```
Comment (ShiChangkuo) [inline huaweicloud/services/dis/resource_huaweicloud_dis_stream.go]: missing logging level

```suggestion
			log.Printf("[DEBUG] WritablePartitionCount: %d, targetValue: %d", resp.WritablePartitionCount, targetValue)
```

File: huaweicloud/services/dis/resource_huaweicloud_dis_stream.go [hunk 4]
```diff
@@ -269,34 +271,34 @@ func resourceDisStreamRead(ctx context.Context, d *schema.ResourceData, meta int
...
 	if err != nil {
-		return diag.Errorf("error creating DIS v2 client, err=%s", err)
+		return diag.Errorf("error creating DIS v2 client, err: %s", err)
 	}
 
 	name := d.Id()
 	errResult := streams.Delete(client, name)
 	if errResult.Err != nil {
-		return fmtp.DiagErrorf("Delete DIS streams failed.stream_name:%s,error:%s", name, errResult.Err)
+		return diag.Errorf("delete DIS streams failed.stream_name:%s,error:%s", name, errResult.Err)
```
Comment (ShiChangkuo) [inline huaweicloud/services/dis/resource_huaweicloud_dis_stream.go]:
```suggestion
		return diag.Errorf("error deleting DIS stream %s: %s", name, errResult.Err)
```

File: huaweicloud/services/dis/resource_huaweicloud_dis_stream.go [hunk 5]
```diff
@@ -217,28 +219,28 @@ func resourceDisStreamCreate(ctx context.Context, d *schema.ResourceData, meta i
...
+	client, err := conf.DisV2Client(region)
 	if err != nil {
-		return diag.Errorf("error creating DIS v2 client, err=%s", err)
+		return diag.Errorf("error creating DIS v2 client, err: %s", err)
 	}
 
 	detail, dErr := streams.Get(client, d.Id(), streams.GetOpts{})
 	if dErr != nil {
-		return fmtp.DiagErrorf("Error query DisStream %q:%s", d.Id(), dErr)
+		return common.CheckDeletedDiag(d, dErr, "error query DisStream")
```
Comment (ShiChangkuo) [inline huaweicloud/services/dis/resource_huaweicloud_dis_stream.go]:
```suggestion
		return common.CheckDeletedDiag(d, dErr, "error query DIS stream")
```

**Pull request merged**